examples/ansible/README.md

```diff
@@ -4,17 +4,27 @@ This playbook will install Sumo Logic Distro of [OpenTelemetry Collector][otc_li
 
 ## Settings
 
-- `sumologic_otel_collector.version`: version of Sumo Logic Distribution for OpenTelemetry Collector
-- `src_config_path`: path to configuration file for Sumo Logic Distribution for OpenTelemetry Collector
-- `memory_high`: defines the throttling limit on memory usage for Sumo Logic Distribution for OpenTelemetry Collector
-- `memory_max`: defines the absolute limit on memory usage for Sumo Logic Distribution for OpenTelemetry Collector
-- `systemd_service`: enables creation of Systemd Service for Sumo Logic Distribution for OpenTelemetry Collector
+- `install_token`: Sumo Logic install token, rel: [install_token]
+- `collector_tags`: Collector tags, these are applied to all processed data
+- `api_url`: Sumo Logic API url. You shouldn't need to set this in most normal circumstances.
+- `version`: version of Sumo Logic Distribution for OpenTelemetry Collector
+- `systemd_service`: enables creation of Systemd Service for Sumo Logic Distribution for OpenTelemetry Collector. Enabled by default. Note that this playbook will not start the collector if you disable this.
+- `src_config_path`: path to configuration directory for Sumo Logic Distribution for OpenTelemetry Collector
 
 ## Running playbook
 
-- Prepare [configuration](../../docs/Configuration.md) for Sumo Logic Distribution for OpenTelemetry Collector
+- Get an install token from Sumo Logic, see [install_token]
+- Prepare [configuration](../../docs/Configuration.md) for Sumo Logic Distribution for OpenTelemetry Collector and put the file in a folder of your choice
```

Comment:
Maybe it is worthy to mention that all files from this directory will be copied to machines where otelcol will be installed.